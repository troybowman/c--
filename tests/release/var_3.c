../decl/var_3.c