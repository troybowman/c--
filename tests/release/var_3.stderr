syntax error, line: 5
syntax error, line: 9
syntax error, line: 13
syntax error, line: 16
syntax error, line: 20
c-- returned a nonzero exit code
